### DISCUSSION TITLE
How to Load Build it map onto Unity?

### DISCUSSION THREAD

**Discussion by K2Ken** [naverz/zepeto-studio-kor#190] [5. Q&A]
Hello could anyone tell me how to load a build it map onto Unity?

I saw the Zepeto Guide and looked around in Unity, but I couldn't find the attachment under Zepeto called Buildit to get the Maploader and run the Buildit Window.

Could somebody please help?

**Comment by beanyh** [comment]
Do you have existing Build it maps?  
Maybe you need to make your own Build it map first and connect (Login) Unity to load maps.

https://studio.zepeto.me/guides/buildit-start

**Reply by K2Ken** [reply]
Yes I do have an existing build it map.

The probelm is I don't know how to get the build it map onto Unity, because the Zepeto Guide doesn't have clear instructions, and I couldn't find the place to load the build it maps onto Unity?

Could anyone tell me the steps of where to go into and how to bring in the Build it maps onto Unity?

**Reply by TheNak976** [reply]
Check my reply ==> https://github.com/naverz/zepeto-studio-kor/discussions/190#discussioncomment-2138851

**Comment by moondory77-naverz** [comment]
Hello, K2Ken.
I think you might looking for this feature..
[Loading Buildit Maps](https://studio.zepeto.me/guides/loading-buildit-maps)
Please check the guide.

**Comment by TheNak976** [comment]
> Hello could anyone tell me how to load a build it map onto Unity?
> 
> I saw the Zepeto Guide and looked around in Unity, but I couldn't find the attachment under Zepeto called Buildit to get the Maploader and run the Buildit Window.
> 
> Could somebody please help?

You have to do this my friend:

Don't forget in package manager to choose **"My Registries"**.

This way you will have all the **"Buildit"** packages

This is the information that **is missing in the official guide**, well the official guide itself **is not very detailed**.

I show you this on the image below

https://i.imgur.com/H4mBY6Y.jpg
